lib/volabit/api/transactions.rb

```diff
@@ -12,16 +12,16 @@ def buy_bitcoins(amount:)
   # Instantly sell bitcoins to get fiat balance.
   # @note The amount is expected in satoshis.
   def sell_bitcoins(amount:)
-    post_to_resource 'api/v1/users/me/sells', {
+    resource :post, 'api/v1/users/me/sells', {
       amount: amount
     }
   end
 
   # Instantly send fiat or bitcoins to an address.
-  # @note The amount is expected in satoshis for bitcoins and cents for fiat
+  # @note The amount is expected in satoshis for bitcoins and cents for
   #       fiat currencies.
-  def send(currency:, amount:, address:)
-    post_to_resource 'api/v1/users/me/send', {
+  def send_money(currency:, amount:, address:)
```

Comment:
If we change method names we should add an alias. If not we have to bumb version to MAJOR